ctpolicy/ctpolicy_test.go

```diff
@@ -102,3 +103,113 @@ func TestLifetimeInMonths(t *testing.T) {
 		})
 	}
 }
+
+func TestGroupByLogs(t *testing.T) {
+	tests := []struct {
+		name      string
+		logGroups map[string]*LogGroupInfo
+		want      map[string]map[string]bool
+	}{
+		{
+			name: "BaseGroup",
+			logGroups: map[string]*LogGroupInfo{
+				BaseName: {
+					name: BaseName,
+					LogURLs: map[string]bool{
+						"ct.googleapis.com/aviator/":   true,
+						"ct.googleapis.com/icarus/":    true,
+						"ct.googleapis.com/rocketeer/": true,
+						"ct.googleapis.com/racketeer/": true,
+						"log.bob.io":                   true,
+					},
+					minInclusions: 2,
+					isBase:        true,
+				},
+			},
+			want: map[string]map[string]bool{
+				"ct.googleapis.com/aviator/": {
+					BaseName: true,
+				},
+				"ct.googleapis.com/icarus/": {
+					BaseName: true,
+				},
+				"ct.googleapis.com/rocketeer/": {
+					BaseName: true,
+				},
+				"ct.googleapis.com/racketeer/": {
+					BaseName: true,
+				},
+				"log.bob.io": {
+					BaseName: true,
+				},
+			},
+		},
+		{
+			name: "ChromeLikeGroups",
+			logGroups: map[string]*LogGroupInfo{
+				"Google-operated": {
+					name: "Google-operated",
+					LogURLs: map[string]bool{
+						"ct.googleapis.com/aviator/":   true,
+						"ct.googleapis.com/icarus/":    true,
+						"ct.googleapis.com/rocketeer/": true,
+						"ct.googleapis.com/racketeer/": true,
+					},
+					minInclusions: 2,
+					isBase:        false,
+				},
+				"Non-Google-operated": {
+					name: "Non-Google-operated",
+					LogURLs: map[string]bool{
+						"log.bob.io": true,
+					},
+					minInclusions: 1,
+					isBase:        false,
+				},
+				BaseName: {
+					name: BaseName,
+					LogURLs: map[string]bool{
+						"ct.googleapis.com/aviator/":   true,
+						"ct.googleapis.com/icarus/":    true,
+						"ct.googleapis.com/rocketeer/": true,
+						"ct.googleapis.com/racketeer/": true,
+						"log.bob.io":                   true,
+					},
+					minInclusions: 2,
+					isBase:        true,
+				},
+			},
+			want: map[string]map[string]bool{
+				"ct.googleapis.com/aviator/": {
+					BaseName:          true,
+					"Google-operated": true,
+				},
+				"ct.googleapis.com/icarus/": {
+					BaseName:          true,
+					"Google-operated": true,
+				},
+				"ct.googleapis.com/rocketeer/": {
+					BaseName:          true,
+					"Google-operated": true,
+				},
+				"ct.googleapis.com/racketeer/": {
+					BaseName:          true,
+					"Google-operated": true,
+				},
+				"log.bob.io": {
+					BaseName:              true,
+					"Non-Google-operated": true,
+				},
+			},
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			got := GroupByLogs(test.logGroups)
+			if !reflect.DeepEqual(got, test.want) {
+				t.Errorf("Unable to map logs to groups properly, got \n%v\n, want \n%v\n", got, test.want)
```

Comment:
style: usual form is to echo the name of the function under test:
```golang
t.Errorf("GroupByLogs()=%v, want %v", got, test.want)
```

Reply:
Changed.